cumulus/polkadot-omni-node/lib/src/nodes/mod.rs

```diff
@@ -22,6 +22,11 @@ use cumulus_primitives_core::ParaId;
 use manual_seal::ManualSealNode;
 use sc_service::{Configuration, TaskManager};
 
+/// The current node version for cumulus official binaries, which takes the basic
+/// SemVer form `<major>.<minor>.<patch>`. It should correspond to the latest
+/// `polkadot` version of a stable release.
+pub const NODE_VERSION: &'static str = "1.17.0";
```

Comment:
I think it could also make sense to have a config var in the root `Cargo.toml` for this and then read it from there in a build script, but if the current process is working then thats fine.

Reply:
Hmm, I think this requires modifying both Cargo.tomls of `polkadot-omni-node` and `polkadot-parachain`, as opposed to a single place that will be used by both. At the same time your suggestion seems cleaner (since it will be hidden for end users, as opposed to expose a public const in `polkadot-omni-node-lib` usable only for official node binaries). Other than this, don't see other benefits (we still need to commit a change to the repo to change the version in Cargo.tomls - maybe the pattern match will be safer).

I'll leave it like this since I consider the benefits aren't that significant.